packages/playground/blueprints/src/lib/steps/enable-multisite.spec.ts

```diff
@@ -25,26 +24,32 @@ describe('Blueprint step enableMultisite', () => {
 	}
 
 	it('should enable a multisite on a scoped URL', async () => {
-		const { php, requestHandler } = await doBootWordPress({
+		const { php } = await doBootWordPress({
 			absoluteUrl: 'http://playground-domain/scope:987987/',
 		});
 		await enableMultisite(php, {});
 
-		const response = await requestHandler.request({
-			url: '/wp-admin/network/',
+		const result = await php.run({
+			code: `
+				<?php
+				echo WP_ALLOW_MULTISITE ? 'true' : 'false';
+			`,
 		});
-		expect(response.text).toContain('My Sites');
-	}, 30_000);
+		expect(result.text).toContain('true');
+	});
 
 	it('should enable a multisite on a scopeless URL', async () => {
-		const { php, requestHandler } = await doBootWordPress({
+		const { php } = await doBootWordPress({
 			absoluteUrl: 'http://playground-domain/',
 		});
 		await enableMultisite(php, {});
 
-		const response = await requestHandler.request({
-			url: '/wp-admin/network/',
+		const result = await php.run({
+			code: `
+				<?php
+				echo WP_ALLOW_MULTISITE ? 'true' : 'false';
+			`,
 		});
-		expect(response.text).toContain('My Sites');
-	}, 30_000);
+		expect(result.text).toContain('true');
```

Comment:
Relying on HTTP requests for unit tests didn't work anymore because of the `wp-login` redirect, so I updated the test to look for this constant instead. 

E2E tests will still do the full check.

Reply:
Your comment would make a great inline comment. This discussion will be lost in backscroll tomorrow, but a comment would still be there two years from now.

It would be useful to change the name of that test to something like "should set the WP_ALLOW_MULTISITE to true" since the old name no longer reflects what it does, e.g. the constant doesn't tell us whether we're actually running a working multisite.

Reply:
I noticed the test name remains unchanged

Reply:
Done [22d7ae8](https://github.com/WordPress/wordpress-playground/pull/1789/commits/22d7ae8d768f4cd10bccda9a911fbb32f995f8f7)